cocos/platform/android/jni/JniHelper.cpp

```diff
@@ -262,17 +262,26 @@ namespace cocos2d {
         if (jstr == nullptr) {
             return "";
         }
-        
+
         JNIEnv *env = JniHelper::getEnv();
         if (!env) {
-            return nullptr;
+            return "";
         }
 
-        const char* chars = env->GetStringUTFChars(jstr, nullptr);
```

Comment:
`GetStringUTFChars` will not deal with `emoticon`. I think it maybe a bug of Android System.